.code-samples.meilisearch.yaml

```diff
@@ -3,6 +3,12 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+get_documents_post_1: |-
+  $client->index('books')->getDocuments((new DocumentsQuery())->setFilter('(rating > 3 AND (genres = Adventure OR genres = Fiction)) AND language = English')->setLimit(3)->setFields(['title', 'genres', 'rating', 'language']));
```

Comment:
The readability isn't great, I'm sure you've tried it on several lines, haven't you? It's that big string too...